### PR TITLE
HTTP-Fib example

### PR DESCRIPTION
**Pull request by brooksmtownsend**
Uploading the example used in the last community meeting, including the VSCode tasks.json. This depends on #48 for inmemory-keyvalue to get published

### CODE REVIEW

File: http-fib/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+[package]
+name = "http-fib"
+version = "0.1.0"
+authors = ["Brooks Townsend <brooksmtownsend@gmail.com>"]
```
Comment (autodidaddict) [inline http-fib/Cargo.toml:4]: might want to switch this to a team name.

File: http-fib/manifest.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    sample: "wasmCloud Fibonacci"
+actors:
+    # Note: we will be specifying this with `wash up --watch`, so it won't be loaded from this manifest
+    - ./target/wasm32-unknown-unknown/debug/http_fib_s.wasm
+capabilities:
+    - image_ref: wasmcloud.azurecr.io/httpserver:0.12.1
+    - image_ref: wasmcloud.azurecr.io/inmemory-keyvalue:0.4.0
+links:
+  # configure the fib actor - httpserver and keyvalue
+  - actor: ${FIB_ACTOR:MDVUB2IKGSH634XUWJSKJPSCLLLUCPOZL3HTYYCE6IC34A42UOHWTQ4P}
```
Comment (autodidaddict) [inline http-fib/manifest.yaml:12]: Can we put a comment at the top of the file explanining what FIB_ACTOR is and how they should use it?

**Review by autodidaddict — CHANGES_REQUESTED** [review]
Just a suggestion to take the personal email address of the manifest

**Comment by brooksmtownsend** [comment]
@autodidaddict bumping this PR, addressed the personal email 👍🏻

**Comment by brooksmtownsend** [comment]
@autodidaddict I won't be upset if we don't want to keep this example but it is ready and composed of actors (no providers to update for OTP runtime) just making sure you didn't close instead of merging on accident.

**Comment by autodidaddict** [comment]
Didn't mean to close this. We can leave this open until it'd upgraded to the latest bits.

**Review by autodidaddict — APPROVED** [review]

**Pull request merged**